release-notes/8.0/preview/api-diff/preview4/Microsoft.AspNetCore.App/8.0-preview4_Microsoft.AspNetCore.Components.Sections.md

```diff
@@ -0,0 +1,10 @@
+# Microsoft.AspNetCore.Components.Sections
+
+``` diff
+ namespace Microsoft.AspNetCore.Components.Sections {
+-    public sealed class SectionOutlet : IComponent, IDisposable, ISectionContentSubscriber
++    [StreamRenderingAttribute(true)]
++    public sealed class SectionOutlet : IComponent, IDisposable, ISectionContentSubscriber
```

Comment:
Is this supposed to produce "compilable" output? Still missing braces and semi-colon.

Reply:
I don't think it's supposed to be compilable. This is a bug in AsmDiff. If it's not too much trouble for you, would you mind adding a github suggestion to fix it whenever you find it?

Reply:
```suggestion
+    public sealed class SectionOutlet : IComponent, IDisposable, ISectionContentSubscriber {
+    }
```
?